npm-shrinkwrap.json

```diff
@@ -2661,6 +3212,12 @@
       "resolved": "https://registry.npmjs.org/slash/-/slash-1.0.0.tgz",
       "dev": true
     },
+    "slice-ansi": {
+      "version": "0.0.4",
+      "from": "slice-ansi@0.0.4",
+      "resolved": "http://127.0.0.1:50473/slice-ansi/-/slice-ansi-0.0.4.tgz",
```

Comment:
Why it is pointing to ``http://127.0.0.1:50473``?
Raising issue while I run ``npm install``